packages/components/src/toolbar-button/stories/index.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import { text } from '@storybook/addon-knobs';
+
+/**
+ * Internal dependencies
+ */
+import { Toolbar } from '../../';
+import ToolbarButton from '../';
+
+export default { title: 'Components/ToolbarButton', component: ToolbarButton };
+
+export const _default = () => {
+	const label = text( 'Label', 'This is an example label.' );
+	const icon = text( 'Icon', 'ellipsis' );
```

Comment:
Can you pick a different icon, please? 😂

Reply:
How about the WordPress logo?

Reply:
💘